Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM quay.io/3scale/system-builder:ruby24
+FROM quay.io/3scale/system-builder:ruby25
```

Comment:
Maybe system-builder for ruby 2.5 should use UBI
So we are testing against the same base image as the productized version

Reply:
https://github.com/3scale/porta/pull/1521#issuecomment-573130323